CHANGELOG.md

```diff
@@ -1,10 +1,14 @@
-## 3.1.1(unreleased)
-Extending support to API 600 with the following HPE OneView resource:
-- oneview_scope
+## 3.2.0(unreleased)
+### Notes
```

Comment:
Do not remove the changelog of the previous unreleased version. Please append it to the current one.

Reply:
Done